It won't fix in the master as it breaks the POSIX.

Index: epm/epm-3.7.patch
===================================================================
RCS file: /cvs/external/epm/epm-3.7.patch,v
retrieving revision 1.11
diff -u -r1.11 epm-3.7.patch
--- epm/epm-3.7.patch	1 Aug 2006 12:20:23 -0000	1.11
+++ epm/epm-3.7.patch	6 Jan 2007 10:10:38 -0000
@@ -247,6 +247,23 @@
 *** misc/epm-3.7/dist.c	Thu Aug  7 16:14:40 2003
 --- misc/build/epm-3.7/dist.c	Wed Jul 26 21:01:11 2006
 ***************
+*** 339,345 ****
+  	   const char *defval)		/* I - Default value of option */
+  {
+    char		*ptr;			/* Pointer to option */
+!   static char	option[256];		/* Copy of file option */
+  
+  
+   /*
+--- 339,345 ----
+  	   const char *defval)		/* I - Default value of option */
+  {
+    char		*ptr;			/* Pointer to option */
+!   static char	option[1024];		/* Copy of file option */
+  
+  
+   /*
+***************
 *** 394,401 ****
     strcpy(platform->machine, "mips");
   #elif defined(__hpux)
@@ -292,6 +309,35 @@
     else if (strncmp(platform->machine, "sun", 3) == 0)
       strcpy(platform->machine, "sparc");
   #endif /* __sgi */
+***************
+*** 550,562 ****
+    char		line[2048],	/* Expanded line from list file */
+  		buf[1024],	/* Original line from list file */
+  		type,		/* File type */
+! 		dst[256],	/* Destination path */
+! 		src[256],	/* Source path */
+! 		pattern[256],	/* Pattern for source files */
+  		user[32],	/* User */
+  		group[32],	/* Group */
+  		*temp,		/* Temporary pointer */
+! 		options[256];	/* File options */
+    int		mode,		/* File permissions */
+  		skip;		/* 1 = skip files, 0 = archive files */
+    dist_t	*dist;		/* Distribution data */
+--- 560,572 ----
+    char		line[2048],	/* Expanded line from list file */
+  		buf[1024],	/* Original line from list file */
+  		type,		/* File type */
+! 		dst[1024],	/* Destination path */
+! 		src[1024],	/* Source path */
+! 		pattern[1024],	/* Pattern for source files */
+  		user[32],	/* User */
+  		group[32],	/* Group */
+  		*temp,		/* Temporary pointer */
+! 		options[1024];	/* File options */
+    int		mode,		/* File permissions */
+  		skip;		/* 1 = skip files, 0 = archive files */
+    dist_t	*dist;		/* Distribution data */
 *** misc/epm-3.7/epm.c	Tue Oct 28 15:48:30 2003
 --- misc/build/epm-3.7/epm.c	Wed Jul 26 20:59:48 2006
 ***************
@@ -304,6 +350,29 @@
     puts("");
     puts("EPM is free software and comes with ABSOLUTELY NO WARRANTY; for details");
     puts("see the GNU General Public License in the file COPYING or at");
+*** misc/epm-3.7/epm.h	Thu Jul 24 10:20:54 2003
+--- misc/build/epm-3.7/epm.h	Sat Jan  6 19:08:44 2007
+***************
+*** 167,175 ****
+    int	mode;				/* Permissions of file */
+    char	user[32],			/* Owner of file */
+  	group[32],			/* Group of file */
+! 	src[512],			/* Source path */
+! 	dst[512],			/* Destination path */
+! 	options[256];			/* File options (nostrip, etc.) */
+  } file_t;
+  
+  typedef struct				/**** Install/Patch/Remove Commands ****/
+--- 167,175 ----
+    int	mode;				/* Permissions of file */
+    char	user[32],			/* Owner of file */
+  	group[32],			/* Group of file */
+! 	src[1024],			/* Source path */
+! 	dst[1024],			/* Destination path */
+! 	options[1024];			/* File options (nostrip, etc.) */
+  } file_t;
+  
+  typedef struct				/**** Install/Patch/Remove Commands ****/
 *** misc/epm-3.7/file.c	Wed Jul 23 23:41:08 2003
 --- misc/build/epm-3.7/file.c	Wed Jul 26 20:59:48 2006
 ***************
